app/mailers/late_reservation_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class LateReservationMailer < ApplicationMailer
+	default from: 'notifications@example.com'
+
+	def forgot_to_buy_gift_email(user)
```

Comment:
I bounce between profs that prefer 2 and 4 space indentation. So tabs are ok, if they are 2 spaces long?